unstructured_inference/models/detectron2onnx.py

```diff
@@ -37,7 +37,7 @@
             "model.onnx",
         ),
         label_map=DEFAULT_LABEL_MAP,
-        confidence_threshold=0.8,
+        confidence_threshold=0.5,
```

Comment:
can we make this configurable via a plain text file/env variable? It feel like we may want to have that ability ourselves at least. And how knows maybe in deployment we want to tweak it for different customers.

Reply:
Yeah, I was thinking about the same, but probably is better to add a new issue to record that is a need